tools/trace_example.txt

```diff
@@ -164,6 +174,12 @@ PID    TID    COMM         FUNC             -
 777    785    automount    SyS_nanosleep    sleep for 500000000 ns
 ^C
 
+The tool automatically adds an `#include` for a few common types, but
+generally you should use the -I argument to have the requisite headers
+files included by the code generator, for compilation to succeed. Use
+either the filename relative to /usr/include, or an absolute path to
+the header file.
```

Comment:
I don't know if this belongs in the examples file. I think it's better suited for the man page.